### DISCUSSION TITLE
WooCommerce Wholesale Prices B2B Plugin

### DISCUSSION THREAD

**Discussion by shubhamrane48** [inspireui/support#5359] [Working on Features]
Wholesale Suite – WooCommerce Wholesale Prices & B2B Plugin (supports both single vendor and multivendor). 

https://wordpress.org/plugins/woocommerce-wholesale-prices/

Please add this feature in single vendor application and multi vendor app.

**Comment by ecomdeveloperbd** [comment]
It's very important feature.
Please implement next version

**Comment by HanielQC** [comment]
https://youtu.be/fgWtlo4lDZ4 - Wholesale is supported by FluxStore 3.8.1 https://1.envato.market/mKdNe - Thank you @all
